geps/gep-91.md

```diff
@@ -16,7 +16,71 @@ This GEP proposes a way to validate the TLS certificate presented by the downstr
 ## Non-Goals
 - Define other fields that can be used to verify the client certificate such as the Cerificate Hash or Subject Alt Name. 
 
+### API
+
+* Introduce a `clientValidation` field with [Gateway.Tls][] that can be used to validate the client intiating the TLS connection
+to the Gateway
+* Introduce a `caCerficateRefs` field within `clientValidation` that can be used to specify a list of CA Certificates that
+can be used as a trusted anchor to validate the certificates presented by the client
+
+#### GO
+
+```go
+// ClientValidationContext holds configuration that can be used to validate the client intiating the TLS connection
+// to the Gateway.
+// By default, no client specific configuration is validated.
+type ClientValidationContext struct {
+    // CACertificateRefs contains one or more references to
+    // Kubernetes objects that contain TLS certificates of
+    // the Certificate Authorities that can be used to
+    // as a trusted anchor to validate the certificates presented by the client.
+    //
+    // A single CACertificateRef to a Kubernetes ConfigMap with a key called `ca.crt`
+    // has "Core" support.
+    // Implementations MAY choose to support attaching multiple certificates to
+    // a Listener, but this behavior is implementation-specific.
+    //
+    // References to a resource in different namespace are invalid UNLESS there
+    // is a ReferenceGrant in the target namespace that allows the certificate
+    // to be attached. If a ReferenceGrant does not allow this reference, the
+    // "ResolvedRefs" condition MUST be set to False for this listener with the
+    // "RefNotPermitted" reason.
```

Comment:
Can you also add docs on the behavior when the CACertificateRefs is empty but needed?  Echoing https://github.com/kubernetes-sigs/gateway-api/pull/2273/files#r1334951299 to some extent.

Reply:
I didn't see an update for this.

Reply:
im not sure I understand this, `caCaertificateRefs` is an optional field within `clientValidation`, if its empty it is assumed that is the intent and it is valid, because in the future `clientValidation` may have more fields within it such as `san`